test/controllers/event_controller_test.exs

```diff
@@ -21,4 +22,10 @@ defmodule Dwylbot.EventTestController do
       assert json_response(conn, 200)
     end
   end
+
+  test "helper wait", _conn do
+      assert Helpers.wait(:prod, 1, 2, 3) == 1
+      assert Helpers.wait(:dev, 1, 2, 3) == 2
+      assert Helpers.wait(:test, 1, 2, 3) == 3
+  end
```

Comment:
You already have a `@doc` test for this function, you shouldn't have both. I'd say get rid of this one and have

```elixir
doctest Dwylbot.Rules.Helpers
```

At the top of this file